lib/Bio/EnsEMBL/DataCheck/Checks/DisplayNameFormat.pm

```diff
@@ -40,7 +40,7 @@ sub tests {
   my $mca = $self->dba->get_adaptor("MetaContainer");
 
   # Check that the format of the display name conforms to expectations.
-  my $format = '[A-Za-z0-9\ ]+ \([A-Za-z0-9\(\)\/\-\_,\#\. ]+\) \- GCA_\d+\.\d+';
+  my $format = '[A-Za-z0-9\ ]+ \([A-Za-z0-9\(\)\/\-\_,\#\. ]+\) \- GCA_\d+\.\d+(?:\s\[[\w ]+\])?';
```

Comment:
do we allow for `:` in  `[]`?

Reply:
Not entirely sure, I am not the one who committed this : https://github.com/Ensembl/ensembl-datacheck/commit/ed3c9aa55fc8deb21cdf2987dbdfb6507ab6c852 :-)

Reply:
sorry. That's for stuff like `[FlyBase annotation]`, etc. Seems to be ok.